src/transformers/integrations/integration_utils.py

```diff
@@ -940,6 +941,7 @@ def __init__(self):
         if not is_mlflow_available():
             raise RuntimeError("MLflowCallback requires mlflow to be installed. Run `pip install mlflow`.")
         import mlflow
+        self._ml_flow_version = importlib.metadata.version('mlflow')
```

Comment:
not sure we need to set this as an attribute no?

Reply:
I changed it to a `self._async_log` attribute instead, to avoid parsing and comparing the version string every time the program logs something.